scripts/population/README.md

```diff
@@ -5,39 +5,51 @@
 
 ### Regions file
 
-* Download former MapZen boundary information and store it as `./data/uruguay_geojson.tgz`
+__Update Fix #285:__
 
-```sh
-$ curl -L https://github.com/instedd/planwise/files/1803300/uruguay_geojson.tar.gz -o ./data/uruguay_geojson.tgz
-```
+See script at `/scripts/geojson` to download KMZ files from [GADM](https://gadm.org/) and convert them to geojson.
+
+__Before:__
```

Comment:
we can remove the old way. we can come to this PR to check that instructions if needed.